examples/client/client.c

```diff
@@ -834,15 +834,17 @@ const char* starttlsCmd[6] = {
 static int StartTLS_Init(SOCKET_T* sockfd)
 {
     char tmpBuf[512];
+    char buf[4];
 
     if (sockfd == NULL)
         return BAD_FUNC_ARG;
-
+    
     /* S: 220 <host> SMTP service ready */
     XMEMSET(tmpBuf, 0, sizeof(tmpBuf));
     if (recv(*sockfd, tmpBuf, sizeof(tmpBuf)-1, 0) < 0)
         err_sys("failed to read STARTTLS command\n");
 
+    XMEMCPY(buf,tmpBuf,XSTRLEN(starttlsCmd[0]));
```

Comment:
Please use spaces after the comma's `, `. Shouldn't the copy include the null terminating character? So use `XSTRLEN() + 1`?